packages/core/src/chains/defaults.ts

```diff
@@ -15,6 +15,7 @@ import {
 
 export interface SDKConfig {
     readonly id: ChainId;
+    readonly name: string;
```

Comment:
I see we use the `name` to select which token mapping to use. Can we call this field "environment", then? "Name" feels like it should totally describe the SDK

Reply:
i see, yeah "environment" or "chainName" is more clear here